parcels/particle.py

```diff
@@ -39,15 +39,24 @@ class Particle(object):
     :param lon: Initial longitude of particle
     :param lat: Initial latitude of particle
     :param grid: :Class Grid: object to track this particle on
+    :param user_vars: Dictionary of any user variables that might be defined in subclasses
     """
+    user_vars = OrderedDict()
 
     def __init__(self, lon, lat, grid, cptr=None):
         self.lon = lon
         self.lat = lat
-
         self.xi = np.where(self.lon >= grid.U.lon)[0][-1]
         self.yi = np.where(self.lat >= grid.U.lat)[0][-1]
 
+        # If particle is a superclass with user variables,
+        # initiate these attributes from the grid here
+        for var in self.user_vars:
+            if hasattr(grid, var):
+                setattr(self, var, getattr(grid, var)[0, lon, lat])
+            else:
+                raise AttributeError('The current grid does contain the "%s" field defined in particle.user_vars.' % var)
```

Comment:
The way this is currently written every `user_var` must exist on the grid to not cause the error. I don't think this is the intended behaviour and it's what causes the current Travis failures, I believe. A better way would probably be to just initialise all `user_vars` to 0 and let the user kernel do the sampling. 

My reasoning here is that automatically inferring a relationship between grid and particle variables solely based on name might cause unexpected behaviour from a users point of view and I don't think adding a single line to make the sampling process explicit is too much to ask. I'm open to discussion on this point though. Also, please note that I do not object to adding a helper kernel `UpdateUserVars` that makes this assumption, but this should be clearly documented.